src/pages/addCars.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import FormError from '../components/Form/FormError';
+import PrimaryButton from '../components/Form/PrimaryButton';
+
+const AddCars = () => {
+  const {
+    register,
+    formState: { errors },
+  } = useForm();
+
+  const [formSuccess] = useState(false);
+
+  // const onSubmit = (data) => {
+  //   const body = { bike: { ...data } };
+  //   dispatch(addBike(body));
+  //   reset();
+  //   setFormSuccess(true);
+  //   setTimeout(() => setFormSuccess(false), 3000);
+  // };
+
+  return (
+    <section className="fixed top-0 w-full h-full md:pl-[9vw] bg-[url('/src/images/car-medium.png')] md:bg-[url('/src/images/2-2-car-transparent.png')] bg-center bg-no-repeat bg-200%">
+      <div className="w-full h-full bg-[#efefef]/90 px-20 flex flex-col justify-center">
+        <div className="h-full text-center flex flex-col items-center pt-20 w-5/5">
+          <h1 className="font-bold text-gray-700 text-2xl md:text-5xl text-center uppercase mb-10">
+            Add Cars
+          </h1>
+          <p
```

Comment:
#### [Optional]

- [ ] The form on the "add cars" seems to overflow the body while it's not scrollable. Kindly check the paddings to fix this